README.md

```diff
@@ -30,27 +30,22 @@ Check out our [docs](https://deepcell-kiosk.readthedocs.io/en/master/GETTING_STA
 
 ## Software Architecture
 
-![Kiosk Architecture](https://raw.githubusercontent.com/vanvalenlab/kiosk/mrgn-docs/docs/images/Kiosk_Architecture.png)
+![Kiosk Architecture](https://raw.githubusercontent.com/vanvalenlab/kiosk/wg-docs-diagram/docs/images/Kiosk_Architecture.png)
 
-<tt><a href="https://github.com/vanvalenlab/kiosk-frontend">kiosk-frontend</a></tt>
+- <a href="https://github.com/vanvalenlab/kiosk-frontend">Frontend</a>: API for creating and managing jobs, and a React-based web interface for the DeepCell Kiosk.
 
-- DeepCell graphical user interface built using React, Babel, Webpack.
+- <a href="https://github.com/vanvalenlab/kiosk-redis-consumer">Consumer</a>: Retrieves items from the Job Queue and handles the processing pipeline for that item. Each consumer only works on one item at a time.
 
-<tt><a href="https://github.com/vanvalenlab/kiosk-redis-consumer">kiosk-redis-consumer</a></tt>
+- <a href="https://github.com/vanvalenlab/kiosk-tf-serving">Model Server</a>: Serves models over a gRPC API, allowing consumers to send data and get back predictions.
 
-- Retrieves items from the queue and handles the processing pipeline for that item. Each consumer handles one item at a time.
+- <a href="https://github.com/vanvalenlab/kiosk-autoscaler">GPU Autoscaler</a>: Automatically and efficiently scales Kubernetes GPU resources.
 
-<tt><a href="https://github.com/vanvalenlab/kiosk-bucket-monitor">kiosk-bucket-monitor</a></tt>
+- _Not pictured above_:
 
-- The `bucket-monitor` will monitor all bucket folders listed in the `PREFIX` environment variable (defaults to `PREFIX: "uploads/,output/"`). Any files that are older than 3 days (configured the `AGE_THRESHOLD` environment variable) will be deleted.
+  * <a href="https://github.com/vanvalenlab/kiosk-bucket-monitor">Bucket Monitor</a>: Purges the bucket of uploaded and processed files that are older than `AGE_THRESHOLD`, 3 days by default.
 
-<tt><a href="https://github.com/vanvalenlab/kiosk-tf-serving">kiosk-tf-serving</a></tt>
+  * <a href="https://github.com/vanvalenlab/kiosk-redis-janitor">Janitor</a>: Monitors in-progress items and makes sure no jobs get left un-finished.
```

Comment:
These links can all be changed back to normal markdown links. The html was only necessary for links that needed to appear as code.